### ISSUE TITLE
Incorrect `MozartTrio` example

### ISSUE THREAD

**Issue by lemzwerg** [w3c/musicxml#568]
[I'm not sure whether this is the right place to report problems with files from `musicxml-samples.zip`]

The rendering as shown in `MozartTrio.png` doesn't reflect the actual contents of `MozartTrio.mxl`.  There is an additional tenuto line shown attached to the fourth eighth note in bar 9 of the topmost staff, which doesn't exist in the MusicXML code.

![image](https://github.com/user-attachments/assets/6f06d46f-e51d-48eb-86f3-068cd4ea0dd3)

Additionally, the position of the bar number counter (digit 8) is too far to the right.

**Comment by mscuthbert** [comment]
Yes and no -- This is as good a place as any to report such things, I guess.  However, the files are property of MakeMusic (as is the musicxml.com site) which have not been officially contributed to the W3C repository.  They're in a bit of a limbo, I'm afraid.  

This image for instance is referenced as coming from the Stanford CCARH website, so it would need to be updated there.

The musicxml file here has `<rights>Copyright © 2010 MakeMusic, Inc.</rights>` so it's not something we can include.  Because of the importance this particular excerpt has had in the history of computer music notation, it would be good to reengrave anew and to release as public domain so it can be included on the W3C MusicXML collection.

**Comment by mscuthbert** [comment]
(Leaving this open for a few weeks, in case anyone wants to take up the task or MakeMusic wants to release the file.  Otherwise will close as no action available)

**Comment by infojunkie** [comment]
> so it can be included on the W3C MusicXML collection

Out of curiosity, where is this collection located? I can't find any sample in this repo, other than the annotated examples.

**Comment by mdgood** [comment]
These sample files are at https://www.musicxml.com/music-in-musicxml/example-set/.

I've looked at that example for almost 25 years and never saw the tenuto there since it coincides with the staff line. You can't see it in the version published in the _Beyond MIDI_ book. It doesn't appear in their MuseData encoding, although there's a strange line break before the following D. It doesn't appear in the Breitkopf & Härtel / Dover version on IMSLP.

So I think the problem is in the image, not the MusicXML file. The images are the property of CCARH so they would need to donate them if we wish to use them here.

The bar number is an engraving quirk in the image. It didn't seem to make sense to duplicate that in the MusicXML code. The goal of the CCARH examples is to show how different format represent semantics, rather than appearance details.

I could certainly ask folks at MakeMusic to contribute the Recordare Editions and Beyond MIDI files. The Contributed Files have a similar issue with permissions from the copyright holders. But due to historical reasons, the Recordare Editions are all art songs. It seems a set of examples from public domain music at the Community Group could be chosen from a much wider range of repertoire.

Perhaps this issue could be closed, and a separate issue/discussion started regarding a set of more complete MusicXML examples to accompany the spec. That seems like a separate issue from a test suite, which could seem to benefit more from smaller, isolated examples of particular encoding features. I thought we had a test suite issue, but I didn't see it in a quick review of open issues.

**Comment by lemzwerg** [comment]
A test suite is [here](https://gitlab.com/lilypond/lilypond/-/tree/master/input/regression/musicxml?ref_type=heads); I'm working almost constantly the last few months to clean up the tests, also adding new ones as my revision of `musicxml2ly` continues.  Hopefully, I'm done soon.

**Comment by infojunkie** [comment]
The Lilypond MusicXML test suite is indeed the most comprehensive set we've got so far. Thanks for this great work.

**Comment by mscuthbert** [comment]
I've been keeping the test suite up to date and expanding it with Reinhold Kainhofer's blessing at https://github.com/cuthbertLab/musicxmlTestSuite  -- trying to do things like consolidating the invalid MusicXML files (but with errors seen in the wild) as Test 99.

**Comment by mscuthbert** [comment]
I'm going to close this as not an issue for the W3C group, but feel free to continue discussion or reopen if there's an actionable item that someone is willing to take on.
